src/base.rs

```diff
@@ -50,7 +50,7 @@ impl InvertableTransform for BaseConversion {
             result += *b as u64;
```

Comment:
Can this (and the *= above still overflow?)

Reply:
I think so, but I'll defer that for now to get this fix in and we can do that in a future PR.